tests/misc/CachingTest.py

```diff
@@ -105,8 +105,8 @@ def test_caching_results(self):
                     coala.main,
                     "coala",
                     "-c", os.devnull,
+                    "--disable-caching",
```

Comment:
Coverage. Nobody should ever need `disable-caching`, but I'm adding it just in case there's an issue with caching